queries.sql

```diff
@@ -15,3 +15,33 @@ SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
 
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+/* Query and update table */
+
+BEGIN TRANSACTION;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
```

Comment:
* I suggest you properly state these insertions `Inside a transaction update the animal's table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that species columns went back to the state before transaction.`  
* You only started the transaction, made the change (` UPDATE animals SET species = 'unspecified';`) but you didn't verify that change was made and undone for example:  ` SELECT species from animals;` 
  ` ROLLBACK;`
Then
  ` SELECT species from animals; `
`

Reply:
Done